src/components/admin/MainIssueController.jsx

```diff
@@ -65,26 +68,33 @@ export default class MainIssueController extends FalcorController {
         this.safeSetState({ changed: changedFlag });
       }
     }, 500);
+
+    this.handleDateChange = (event, date) => {
+      this.safeSetState({ published_at: date });
+    };
+    this.disableDate = this.disableDate.bind(this);
```

Comment:
unnecessary as it doesn't use this

Reply:
which part are you referring to? I call both at some point in the DatePicker

Reply:
disableDate, as it doesn't use `this`, it doesn't need the context